images/calico-accountant/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Update Alpine base system without recompiling calico-accountant
+# docker build -f Dockerfile.updatebase -t ghcr.io/elastisys/calico-accountant:0.1.6.ck8s${N++}
+FROM ghcr.io/elastisys/calico-accountant:0.1.6 as previous
+
+FROM alpine
+WORKDIR /root/
+RUN apk --update add iptables
```

Comment:
Does this install the nftables variants as well?

Reply:
`nft` command line? No. Assuming the older image was built from [this Dockerfile](https://github.com/monzo/calico-accountant/blob/master/Dockerfile) then it doesn't either.

```console
~ # apk list -I
alpine-baselayout-3.4.3-r1 x86_64 {alpine-baselayout} (GPL-2.0-only) [installed]
alpine-baselayout-data-3.4.3-r1 x86_64 {alpine-baselayout} (GPL-2.0-only) [installed]
alpine-keys-2.4-r1 x86_64 {alpine-keys} (MIT) [installed]
apk-tools-2.14.0-r0 x86_64 {apk-tools} (GPL-2.0-only) [installed]
busybox-1.36.0-r9 x86_64 {busybox} (GPL-2.0-only) [installed]
busybox-binsh-1.36.0-r9 x86_64 {busybox} (GPL-2.0-only) [installed]
ca-certificates-bundle-20230506-r0 x86_64 {ca-certificates} (MPL-2.0 AND MIT) [installed]
iptables-1.8.9-r2 x86_64 {iptables} (GPL-2.0-or-later) [installed]
libc-utils-0.7.2-r5 x86_64 {libc-dev} (BSD-2-Clause AND BSD-3-Clause) [installed]
libcrypto3-3.1.0-r4 x86_64 {openssl} (Apache-2.0) [installed]
libmnl-1.0.5-r1 x86_64 {libmnl} (LGPL-2.1-or-later) [installed]
libnftnl-1.2.5-r1 x86_64 {libnftnl} (GPL-2.0-or-later) [installed]
libssl3-3.1.0-r4 x86_64 {openssl} (Apache-2.0) [installed]
musl-1.2.4-r0 x86_64 {musl} (MIT) [installed]
musl-utils-1.2.4-r0 x86_64 {musl} (MIT AND BSD-2-Clause AND GPL-2.0-or-later) [installed]
scanelf-1.3.7-r1 x86_64 {pax-utils} (GPL-2.0-only) [installed]
ssl_client-1.36.0-r9 x86_64 {busybox} (GPL-2.0-only) [installed]
zlib-1.2.13-r1 x86_64 {zlib} (Zlib) [installed]
```

Reply:
Not `nft` on its own but `iptables-nft` variants, including restore and save and matching ipv6 versions. But it seems to be included so it is good to go.